jekyll/_cci2/parallelism-faster-jobs.md

```diff
@@ -27,47 +27,7 @@ CircleCI test splitting lets you intelligently define where splits happen across
 * By **size**
 * Using **timing** data
 
-## How test splitting works
-{: #how-test-splitting-works }
-
-Using **timing-based** test splitting as an example, timing data from the _previous_ test run is used to split a test suite as evenly as possible over a specified number of test environments running in parallel. This delivers the lowest possible test time for the compute power in use.
-
-![Test Splitting]({{ site.baseurl }}/assets/img/docs/test_splitting.png)
-
-Timings-based test splitting gives the most accurate split, and is guaranteed to optimize with each test suite run. The most recent timings data is always used to define where splits are made.
-
-As an example, take a Go test suite. Here, all tests run sequentially in a single test environment, a Docker container:
-
-```yaml
-jobs:
-  build:
-    docker:
-      - image: cimg/go:1.18.1
-    resource_class: large
-    working_directory: ~/my-app
-    steps:
-      - run: go test
-```
-
-To split these tests using timing data:
-
-1. Introduce parallelism to spin up a number of identical test environments (10 in this example) 
-2. Use the `circleci tests split` command, with the `--split-by=timings` flag to split the tests evenly across all executors.
-
-```yaml
-jobs:
-  build:
-    docker:
-      - image: cimg/go:1.18.1
-    parallelism: 10
-    resource_class: large
-    working_directory: ~/my-app
-    steps:
-      - run: go test -v $(go list ./... | circleci tests split --split-by=timings)
-```
-
-The first time the tests are run there will be no timing data for the command to use, but on subsequent runs the test time will be optimized.
-{: class="alert alert-info"}
+It is also possible to use the CLI to [manually allocate tests](#manual-allocation) across parallel environments. Another alternative is to use [environment variables](#using-environment-variables-to-split-tests) instead of the CLI to configure split tests.
```

Comment:
Is this an internal page link reference? I don't think the #use-environment-variables-to-split-tests section exists on this page anymore.